hanoying_back/GameState state # current state of the game
---
hanoying_back/GameMoveGoal[] moves # move to execute to solve